pkgs/applications/misc/electrum/default.nix

```diff
@@ -102,7 +104,8 @@ python3Packages.buildPythonApplication rec {
                 "Exec=$out/bin/electrum %u" \
       --replace 'Exec=sh -c "PATH=\"\\$HOME/.local/bin:\\$PATH\"; electrum --testnet %u"' \
                 "Exec=$out/bin/electrum --testnet %u"
-  '';
+
+  '' + (if enableQt then "wrapQtApp $out/bin/electrum" else "");
```

Comment:
I think this should really happen in the `postFixup` phase

```
postFixup = lib.optionalString enableQt ''
  wrapQtApp $out/bin/electrum
'';
```

Reply:
@worldofpeace, please feel free to add a commit with that improvement.

Reply:
Done.